cmd/dlv/dlv_test.go

```diff
@@ -1119,10 +1119,6 @@ func TestVersion(t *testing.T) {
 }
 
 func TestStaticcheck(t *testing.T) {
-	if goversion.VersionAfterOrEqual(runtime.Version(), 1, 18) {
```

Comment:
This needs to stay because staticceck is still failing, I don't think there's a version of staticcheck that supports 1.18 yet.

Reply:
PS. I doesn't affect test coverage because we are still running staticcheck on the 1.17 and 1.16 builders, we could wait until 1.20 to remove this.